### ISSUE TITLE
Blender Addon - Blender 3.0 

### ISSUE THREAD

**Issue by ASPePeX** [FUSEEProjectTeam/Fusee#388]
Our export addon crashes when loading in Blender 3.0

```
Read prefs: C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\config\userpref.blend
Traceback (most recent call last):
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\FusSceneWriter.py", line 8, in <module>
    from proto import FusSerialization_pb2 as FusSer
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\proto\FusSerialization_pb2.py", line 20, in <module>
    _DISTRIBUTION = DESCRIPTOR.enum_types_by_name['Distribution']
AttributeError: 'NoneType' object has no attribute 'enum_types_by_name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\FusSceneWriter.py", line 15, in <module>
    import FusSerialization_pb2 as FusSer
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\proto\FusSerialization_pb2.py", line 20, in <module>
    _DISTRIBUTION = DESCRIPTOR.enum_types_by_name['Distribution']
AttributeError: 'NoneType' object has no attribute 'enum_types_by_name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "G:\Games\SteamLibrary\steamapps\common\Blender\3.0\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\__init__.py", line 50, in <module>
    from .BlenderVisitor import BlenderVisitor
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\BlenderVisitor.py", line 2, in <module>
    from FusSceneWriter import FusSceneWriter
  File "C:\Users\ASPePeX\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\io_export_fus\FusSceneWriter.py", line 17, in <module>
    print('Error importing "FusSerialization_pb2.py" from "' + dir_path + '"' + '\n' + ex)
TypeError: can only concatenate str (not "AttributeError") to str
```

**Comment by griestopf** [comment]
Reproducable here

**Comment by griestopf** [comment]
Same problem seems to occur with Blender 2.93

Problem does NOT occur when directly accessing Add-On sources from Fusee Repo as described in https://fusee3d.org/wiki/FUSEE-Exporter-Blender-Add-on.html#debugging. This way, Add-On works as expected in Blender 3

Diff between Add-on as downloaded from [Release](https://github.com/FUSEEProjectTeam/Fusee/releases/latest/download/io_export_fus.zip) (left) and Add-on in Repo-src (right):

![Addondiff](https://user-images.githubusercontent.com/1692092/144868620-43d09211-6643-4846-bc91-9284ba6e64a6.PNG)

Not sure where Blender 3 Python takes the google python code from when started from the Repo

**Comment by griestopf** [comment]
According to the messages above the error occurs in io_export_fus\proto\FusSerialization_pb2.py, line 20. This file is generated during build using the google protobuf "compiler" from a custom build step in [Fusee.Tools.CmdLine.csproj](https://github.com/FUSEEProjectTeam/Fusee/blob/master/src/Tools/CmdLine/Fusee.Tools.CmdLine.csproj#L89). This is for Windows... there's a slightly different [command for Linux](https://github.com/FUSEEProjectTeam/Fusee/blob/master/src/Tools/CmdLine/Fusee.Tools.CmdLine.csproj#L116)

As a matter of fact, the FusSerialization_pb2.py generated on my Dev machine is different from the one bundled in the [downloadable Add-on io_export_fus.zip](https://github.com/FUSEEProjectTeam/Fusee/releases/tag/v0.11.2).

Here are the first 23 lines of io_export_fus\proto\FusSerialization_pb2.py generated from Develop on my Dev machine:
***GOOD***
```python
# -*- coding: utf-8 -*-
# Generated by the protocol buffer compiler.  DO NOT EDIT!
# source: FusSerialization.proto

from google.protobuf.internal import enum_type_wrapper
from google.protobuf import descriptor as _descriptor
from google.protobuf import message as _message
from google.protobuf import reflection as _reflection
from google.protobuf import symbol_database as _symbol_database
# @@protoc_insertion_point(imports)

_sym_db = _symbol_database.Default()




DESCRIPTOR = _descriptor.FileDescriptor(
  name='FusSerialization.proto',
  package='',
  syntax='proto3',
  serialized_options=None,
  serialized_pb=b'\n\x16\x46usSerialization.proto\"3\n\x05\x41\x41\x42\x42\x66\x12\x14\n\x03min\x18\x01 \x01(\x0b\x32\x07.float3\x12\x14\n\x03max\x18\x02 \x01(\x0b\x32\x07.float3\"}\n\rAlbedoChannel\x12\x16\n\x05\x43olor\x18\x01 \x01(\x0b\x32\x07.float4\x12\x0f\n\x07Texture\x18\x02 \x01(\t\x12\x0b\n\x03Mix\x18\x03 \x01(\x02\x12+\n\x0fSpecularChannel\x18\x64 \x01(\x0b\x32\x10.SpecularChannelH\x00\x42\t\n\x07subtype\"\x87\x01\n\x0b\x42RDFChannel\x12\x0b\n\x03IOR\x18\x01 \x01(\x02\x12\x10\n\x08Metallic\x18\x02 \x01(\x02\x12\x10\n\x08Specular\x18\x03 \x01(\x02\x12\x11\n\tRoughness\x18\x04 \x01(\x02\x12\x12\n\nSubsurface\x18\x05 \x01(\x02\x12 \n\x0fSubsurfaceColor\x18\x06 \x01(\x0b\x32\x07.float3\";\n\x0c\x46usAnimation\x12+\n\x0f\x41nimationTracks\x18\x03 \x03(\x0b\x32\x12.FusAnimationTrack\"\xfc\x02\n\x13\x46usAnimationKeyBase\x12\x0c\n\x04Time\x18\x01 \x01(\x02\x12\x37\n\x15\x46usAnimationKeyDouble\x18\x64 \x01(\x0b\x32\x16.FusAnimationKeyDoubleH\x00\x12\x31\n\x12\x46usAnimationKeyInt\x18\x65 \x01(\x0b\x32\x13.FusAnimationKeyIntH\x00\x12\x35\n\x14\x46usAnimationKeyFloat\x18\x66 \x01(\x0b\x32\x15.FusAnimationKeyFloatH\x00\x12\x37\n\x15\x46usAnimationKeyFloat2\x18g \x01(\x0b\x32\x16.FusAnimationKeyFloat2H\x00\x12\x37\n\x15\x46usAnimationKeyFloat3\x18h \x01(\x0b\x32\x16.FusAnimationKeyFloat3H\x00\x12\x37\n\x15\x46usAnimationKeyFloat4\x18i \x01(\x0b\x32\x16.FusAnimationKeyFloat4H\x00\x42\t\n\x07subtype\"&\n\x15\x46usAnimationKeyDouble\x12\r\n\x05Value\x18\x01 \x01(\x01\"%\n\x14\x46usAnimationKeyFloat\x12\r\n\x05Value\x18\x01 \x01(\x02\"/\n\x15\x46usAnimationKeyFloat2\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float2\"/\n\x15\x46usAnimationKeyFloat3\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float3\"/\n\x15\x46usAnimationKeyFloat4\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float4\"#\n\x12\x46usAnimationKeyInt\x12\r\n\x05Value\x18\x01 \x01(\x05\"\xab\x01\n\x11\x46usAnimationTrack\x12%\n\x0eSceneComponent\x18\x01 \x01(\x0b\x32\r.FusComponent\x12\x10\n\x08Property\x18\x02 \x01(\t\x12\x17\n\x06TypeId\x18\x03 \x01(\x0e\x32\x07.TypeId\x12\'\n\tKeyFrames\x18\x04 \x03(\x0b\x32\x14.FusAnimationKeyBase\x12\x1b\n\x08LerpType\x18\x05 \x01(\x0e\x32\t.LerpType\"\x17\n\x07\x46usBone\x12\x0c\n\x04Name\x18\x01 \x01(\t\"\xea\x01\n\tFusCamera\x12\x12\n\nClearColor\x18\x01 \x01(\x08\x12\x12\n\nClearDepth\x18\x02 \x01(\x08\x12\r\n\x05Layer\x18\x03 \x01(\x05\x12 \n\x0f\x42\x61\x63kgroundColor\x18\x04 \x01(\x0b\x32\x07.float4\x12+\n\x10ProjectionMethod\x18\x05 \x01(\x0e\x32\x11.ProjectionMethod\x12\x0b\n\x03\x46ov\x18\x06 \x01(\x02\x12\x1f\n\x0e\x43lippingPlanes\x18\x07 \x01(\x0b\x32\x07.float2\x12\x19\n\x08Viewport\x18\x08 \x01(\x0b\x32\x07.float4\x12\x0e\n\x06\x41\x63tive\x18\t \x01(\x08\"\xbd\x02\n\x0c\x46usComponent\x12\x0c\n\x04Name\x18\x01 \x01(\t\x12%\n\x0c\x46usTransform\x18\x64 \x01(\x0b\x32\r.FusTransformH\x00\x12\x1b\n\x07\x46usMesh\x18\x65 \x01(\x0b\x32\x08.FusMeshH\x00\x12+\n\x0f\x46usMaterialBase\x18\x66 \x01(\x0b\x32\x10.FusMaterialBaseH\x00\x12\x1d\n\x08\x46usLight\x18g \x01(\x0b\x32\t.FusLightH\x00\x12\x1f\n\tFusWeight\x18h \x01(\x0b\x32\n.FusWeightH\x00\x12%\n\x0c\x46usAnimation\x18i \x01(\x0b\x32\r.FusAnimationH\x00\x12\x1b\n\x07\x46usBone\x18j \x01(\x0b\x32\x08.FusBoneH\x00\x12\x1f\n\tFusCamera\x18k \x01(\x0b\x32\n.FusCameraH\x00\x42\t\n\x07subtype\"8\n\x0b\x46usContents\x12\x1e\n\x08\x46usScene\x18\xc9\x01 \x01(\x0b\x32\t.FusSceneH\x00\x42\t\n\x07subtype\"E\n\x07\x46usFile\x12\x1a\n\x06Header\x18\x01 \x01(\x0b\x32\n.FusHeader\x12\x1e\n\x08\x43ontents\x18\x02 \x01(\x0b\x32\x0c.FusContents\"\\\n\tFusHeader\x12\x13\n\x0b\x46ileVersion\x18\x01 \x01(\x05\x12\x11\n\tGenerator\x18\x02 \x01(\t\x12\x11\n\tCreatedBy\x18\x03 \x01(\t\x12\x14\n\x0c\x43reationDate\x18\x04 \x01(\t\"\xa3\x01\n\x08\x46usLight\x12\x0e\n\x06\x41\x63tive\x18\x01 \x01(\x08\x12\x16\n\x05\x43olor\x18\x02 \x01(\x0b\x32\x07.float4\x12\x13\n\x0bMaxDistance\x18\x03 \x01(\x02\x12\x10\n\x08Strength\x18\x04 \x01(\x02\x12\x18\n\x04Type\x18\x05 \x01(\x0e\x32\n.LightType\x12\x16\n\x0eOuterConeAngle\x18\x06 \x01(\x02\x12\x16\n\x0eInnerConeAngle\x18\x07 \x01(\x02\"-\n\x0f\x46usMaterialBRDF\x12\x1a\n\x04\x42RDF\x18\x02 \x01(\x0b\x32\x0c.BRDFChannel\"\xde\x02\n\x0f\x46usMaterialBase\x12\x1e\n\x06\x41lbedo\x18\x01 \x01(\x0b\x32\x0e.AlbedoChannel\x12 \n\x08\x45missive\x18\x03 \x01(\x0b\x32\x0e.AlbedoChannel\x12$\n\tNormalMap\x18\x04 \x01(\x0b\x32\x11.NormapMapChannel\x12\x34\n\x13\x46usMaterialStandard\x18\xc8\x01 \x01(\x0b\x32\x14.FusMaterialStandardH\x00\x12,\n\x0f\x46usMaterialBRDF\x18\xc9\x01 \x01(\x0b\x32\x10.FusMaterialBRDFH\x00\x12:\n\x16\x46usMaterialDiffuseBRDF\x18\xca\x01 \x01(\x0b\x32\x17.FusMaterialDiffuseBRDFH\x00\x12\x38\n\x15\x46usMaterialGlossyBRDF\x18\xcb\x01 \x01(\x0b\x32\x16.FusMaterialGlossyBRDFH\x00\x42\t\n\x07subtype\"+\n\x16\x46usMaterialDiffuseBRDF\x12\x11\n\tRoughness\x18\x01 \x01(\x02\"O\n\x15\x46usMaterialGlossyBRDF\x12\x11\n\tRoughness\x18\x01 \x01(\x02\x12#\n\x0c\x44istribution\x18\x02 \x01(\x0e\x32\r.Distribution\"7\n\x13\x46usMaterialStandard\x12 \n\x08Specular\x18\x02 \x01(\x0b\x32\x0e.AlbedoChannel\"\xa2\x02\n\x07\x46usMesh\x12\x19\n\x08Vertices\x18\x01 \x03(\x0b\x32\x07.float3\x12\x12\n\x06\x43olors\x18\x02 \x03(\rB\x02\x10\x00\x12\x18\n\x07Normals\x18\x03 \x03(\x0b\x32\x07.float3\x12\x14\n\x03UVs\x18\x04 \x03(\x0b\x32\x07.float2\x12\x1c\n\x0b\x42oneWeights\x18\x05 \x03(\x0b\x32\x07.float4\x12\x1c\n\x0b\x42oneIndices\x18\x06 \x03(\x0b\x32\x07.float4\x12\x15\n\tTriangles\x18\x07 \x03(\rB\x02\x10\x00\x12\x1b\n\x0b\x42oundingBox\x18\x08 \x01(\x0b\x32\x06.AABBf\x12\x19\n\x08Tangents\x18\t \x03(\x0b\x32\x07.float4\x12\x1b\n\nBiTangents\x18\n \x03(\x0b\x32\x07.float3\x12\x10\n\x08MeshType\x18\x0b \x01(\x05\"K\n\x07\x46usNode\x12\x0c\n\x04Name\x18\x01 \x01(\t\x12\x16\n\nComponents\x18\x02 \x03(\x05\x42\x02\x10\x00\x12\x1a\n\x08\x43hildren\x18\x03 \x03(\x0b\x32\x08.FusNode\"L\n\x08\x46usScene\x12$\n\rComponentList\x18\x01 \x03(\x0b\x32\r.FusComponent\x12\x1a\n\x08\x43hildren\x18\x02 \x03(\x0b\x32\x08.FusNode\"_\n\x0c\x46usTransform\x12\x1c\n\x0bTranslation\x18\x01 \x01(\x0b\x32\x07.float3\x12\x19\n\x08Rotation\x18\x02 \x01(\x0b\x32\x07.float3\x12\x16\n\x05Scale\x18\x03 \x01(\x0b\x32\x07.float3\"t\n\tFusWeight\x12$\n\tWeightMap\x18\x01 \x03(\x0b\x32\x11.VertexWeightList\x12\x1d\n\x06Joints\x18\x02 \x03(\x0b\x32\r.FusComponent\x12\"\n\x0f\x42indingMatrices\x18\x03 \x03(\x0b\x32\t.float4x4\"6\n\x10NormapMapChannel\x12\x0f\n\x07Texture\x18\x01 \x01(\t\x12\x11\n\tIntensity\x18\x02 \x01(\x02\"6\n\x0fSpecularChannel\x12\x11\n\tShininess\x18\x01 \x01(\x02\x12\x10\n\x08Strength\x18\x02 \x01(\x02\"2\n\x0cVertexWeight\x12\x12\n\nJointIndex\x18\x01 \x01(\x05\x12\x0e\n\x06Weight\x18\x02 \x01(\x02\"8\n\x10VertexWeightList\x12$\n\rVertexWeights\x18\x01 \x03(\x0b\x32\r.VertexWeight\"\x1e\n\x06\x66loat2\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\")\n\x06\x66loat3\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\x12\t\n\x01z\x18\x03 \x01(\x02\"4\n\x06\x66loat4\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\x12\t\n\x01z\x18\x03 \x01(\x02\x12\t\n\x01w\x18\x04 \x01(\x02\"f\n\x08\x66loat4x4\x12\x15\n\x04Row1\x18\x01 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row2\x18\x02 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row3\x18\x03 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row4\x18\x04 \x01(\x0b\x32\x07.float4*\x17\n\x0c\x44istribution\x12\x07\n\x03GGX\x10\x00*\x1f\n\x08LerpType\x12\x08\n\x04Lerp\x10\x00\x12\t\n\x05Slerp\x10\x01*:\n\tLightType\x12\t\n\x05Point\x10\x00\x12\x0c\n\x08Parallel\x10\x01\x12\x08\n\x04Spot\x10\x02\x12\n\n\x06Legacy\x10\x03*5\n\x10ProjectionMethod\x12\x0f\n\x0bPerspective\x10\x00\x12\x10\n\x0cOrthographic\x10\x01*V\n\x06TypeId\x12\n\n\x06\x44ouble\x10\x00\x12\x07\n\x03Int\x10\x01\x12\t\n\x05\x46loat\x10\x02\x12\n\n\x06\x46loat2\x10\x03\x12\n\n\x06\x46loat3\x10\x04\x12\n\n\x06\x46loat4\x10\x05\x12\x08\n\x04\x42ool\x10\x06\x62\x06proto3'
)
```

In contrast, here are the first 23 lines of the downloaded o_export_fus\proto\FusSerialization_pb2.py generated on the build server (on Linux?)
***BAD***

```python
# -*- coding: utf-8 -*-
# Generated by the protocol buffer compiler.  DO NOT EDIT!
# source: FusSerialization.proto
"""Generated protocol buffer code."""
from google.protobuf.internal import enum_type_wrapper
from google.protobuf import descriptor as _descriptor
from google.protobuf import descriptor_pool as _descriptor_pool
from google.protobuf import message as _message
from google.protobuf import reflection as _reflection
from google.protobuf import symbol_database as _symbol_database
# @@protoc_insertion_point(imports)

_sym_db = _symbol_database.Default()




DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n\x16\x46usSerialization.proto\"3\n\x05\x41\x41\x42\x42\x66\x12\x14\n\x03min\x18\x01 \x01(\x0b\x32\x07.float3\x12\x14\n\x03max\x18\x02 \x01(\x0b\x32\x07.float3\"}\n\rAlbedoChannel\x12\x16\n\x05\x43olor\x18\x01 \x01(\x0b\x32\x07.float4\x12\x0f\n\x07Texture\x18\x02 \x01(\t\x12\x0b\n\x03Mix\x18\x03 \x01(\x02\x12+\n\x0fSpecularChannel\x18\x64 \x01(\x0b\x32\x10.SpecularChannelH\x00\x42\t\n\x07subtype\"\x87\x01\n\x0b\x42RDFChannel\x12\x0b\n\x03IOR\x18\x01 \x01(\x02\x12\x10\n\x08Metallic\x18\x02 \x01(\x02\x12\x10\n\x08Specular\x18\x03 \x01(\x02\x12\x11\n\tRoughness\x18\x04 \x01(\x02\x12\x12\n\nSubsurface\x18\x05 \x01(\x02\x12 \n\x0fSubsurfaceColor\x18\x06 \x01(\x0b\x32\x07.float3\";\n\x0c\x46usAnimation\x12+\n\x0f\x41nimationTracks\x18\x03 \x03(\x0b\x32\x12.FusAnimationTrack\"\xfc\x02\n\x13\x46usAnimationKeyBase\x12\x0c\n\x04Time\x18\x01 \x01(\x02\x12\x37\n\x15\x46usAnimationKeyDouble\x18\x64 \x01(\x0b\x32\x16.FusAnimationKeyDoubleH\x00\x12\x31\n\x12\x46usAnimationKeyInt\x18\x65 \x01(\x0b\x32\x13.FusAnimationKeyIntH\x00\x12\x35\n\x14\x46usAnimationKeyFloat\x18\x66 \x01(\x0b\x32\x15.FusAnimationKeyFloatH\x00\x12\x37\n\x15\x46usAnimationKeyFloat2\x18g \x01(\x0b\x32\x16.FusAnimationKeyFloat2H\x00\x12\x37\n\x15\x46usAnimationKeyFloat3\x18h \x01(\x0b\x32\x16.FusAnimationKeyFloat3H\x00\x12\x37\n\x15\x46usAnimationKeyFloat4\x18i \x01(\x0b\x32\x16.FusAnimationKeyFloat4H\x00\x42\t\n\x07subtype\"&\n\x15\x46usAnimationKeyDouble\x12\r\n\x05Value\x18\x01 \x01(\x01\"%\n\x14\x46usAnimationKeyFloat\x12\r\n\x05Value\x18\x01 \x01(\x02\"/\n\x15\x46usAnimationKeyFloat2\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float2\"/\n\x15\x46usAnimationKeyFloat3\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float3\"/\n\x15\x46usAnimationKeyFloat4\x12\x16\n\x05Value\x18\x01 \x01(\x0b\x32\x07.float4\"#\n\x12\x46usAnimationKeyInt\x12\r\n\x05Value\x18\x01 \x01(\x05\"\xab\x01\n\x11\x46usAnimationTrack\x12%\n\x0eSceneComponent\x18\x01 \x01(\x0b\x32\r.FusComponent\x12\x10\n\x08Property\x18\x02 \x01(\t\x12\x17\n\x06TypeId\x18\x03 \x01(\x0e\x32\x07.TypeId\x12\'\n\tKeyFrames\x18\x04 \x03(\x0b\x32\x14.FusAnimationKeyBase\x12\x1b\n\x08LerpType\x18\x05 \x01(\x0e\x32\t.LerpType\"\x17\n\x07\x46usBone\x12\x0c\n\x04Name\x18\x01 \x01(\t\"\xea\x01\n\tFusCamera\x12\x12\n\nClearColor\x18\x01 \x01(\x08\x12\x12\n\nClearDepth\x18\x02 \x01(\x08\x12\r\n\x05Layer\x18\x03 \x01(\x05\x12 \n\x0f\x42\x61\x63kgroundColor\x18\x04 \x01(\x0b\x32\x07.float4\x12+\n\x10ProjectionMethod\x18\x05 \x01(\x0e\x32\x11.ProjectionMethod\x12\x0b\n\x03\x46ov\x18\x06 \x01(\x02\x12\x1f\n\x0e\x43lippingPlanes\x18\x07 \x01(\x0b\x32\x07.float2\x12\x19\n\x08Viewport\x18\x08 \x01(\x0b\x32\x07.float4\x12\x0e\n\x06\x41\x63tive\x18\t \x01(\x08\"\xbd\x02\n\x0c\x46usComponent\x12\x0c\n\x04Name\x18\x01 \x01(\t\x12%\n\x0c\x46usTransform\x18\x64 \x01(\x0b\x32\r.FusTransformH\x00\x12\x1b\n\x07\x46usMesh\x18\x65 \x01(\x0b\x32\x08.FusMeshH\x00\x12+\n\x0f\x46usMaterialBase\x18\x66 \x01(\x0b\x32\x10.FusMaterialBaseH\x00\x12\x1d\n\x08\x46usLight\x18g \x01(\x0b\x32\t.FusLightH\x00\x12\x1f\n\tFusWeight\x18h \x01(\x0b\x32\n.FusWeightH\x00\x12%\n\x0c\x46usAnimation\x18i \x01(\x0b\x32\r.FusAnimationH\x00\x12\x1b\n\x07\x46usBone\x18j \x01(\x0b\x32\x08.FusBoneH\x00\x12\x1f\n\tFusCamera\x18k \x01(\x0b\x32\n.FusCameraH\x00\x42\t\n\x07subtype\"8\n\x0b\x46usContents\x12\x1e\n\x08\x46usScene\x18\xc9\x01 \x01(\x0b\x32\t.FusSceneH\x00\x42\t\n\x07subtype\"E\n\x07\x46usFile\x12\x1a\n\x06Header\x18\x01 \x01(\x0b\x32\n.FusHeader\x12\x1e\n\x08\x43ontents\x18\x02 \x01(\x0b\x32\x0c.FusContents\"\\\n\tFusHeader\x12\x13\n\x0b\x46ileVersion\x18\x01 \x01(\x05\x12\x11\n\tGenerator\x18\x02 \x01(\t\x12\x11\n\tCreatedBy\x18\x03 \x01(\t\x12\x14\n\x0c\x43reationDate\x18\x04 \x01(\t\"\xa3\x01\n\x08\x46usLight\x12\x0e\n\x06\x41\x63tive\x18\x01 \x01(\x08\x12\x16\n\x05\x43olor\x18\x02 \x01(\x0b\x32\x07.float4\x12\x13\n\x0bMaxDistance\x18\x03 \x01(\x02\x12\x10\n\x08Strength\x18\x04 \x01(\x02\x12\x18\n\x04Type\x18\x05 \x01(\x0e\x32\n.LightType\x12\x16\n\x0eOuterConeAngle\x18\x06 \x01(\x02\x12\x16\n\x0eInnerConeAngle\x18\x07 \x01(\x02\"-\n\x0f\x46usMaterialBRDF\x12\x1a\n\x04\x42RDF\x18\x02 \x01(\x0b\x32\x0c.BRDFChannel\"\xde\x02\n\x0f\x46usMaterialBase\x12\x1e\n\x06\x41lbedo\x18\x01 \x01(\x0b\x32\x0e.AlbedoChannel\x12 \n\x08\x45missive\x18\x03 \x01(\x0b\x32\x0e.AlbedoChannel\x12$\n\tNormalMap\x18\x04 \x01(\x0b\x32\x11.NormapMapChannel\x12\x34\n\x13\x46usMaterialStandard\x18\xc8\x01 \x01(\x0b\x32\x14.FusMaterialStandardH\x00\x12,\n\x0f\x46usMaterialBRDF\x18\xc9\x01 \x01(\x0b\x32\x10.FusMaterialBRDFH\x00\x12:\n\x16\x46usMaterialDiffuseBRDF\x18\xca\x01 \x01(\x0b\x32\x17.FusMaterialDiffuseBRDFH\x00\x12\x38\n\x15\x46usMaterialGlossyBRDF\x18\xcb\x01 \x01(\x0b\x32\x16.FusMaterialGlossyBRDFH\x00\x42\t\n\x07subtype\"+\n\x16\x46usMaterialDiffuseBRDF\x12\x11\n\tRoughness\x18\x01 \x01(\x02\"O\n\x15\x46usMaterialGlossyBRDF\x12\x11\n\tRoughness\x18\x01 \x01(\x02\x12#\n\x0c\x44istribution\x18\x02 \x01(\x0e\x32\r.Distribution\"7\n\x13\x46usMaterialStandard\x12 \n\x08Specular\x18\x02 \x01(\x0b\x32\x0e.AlbedoChannel\"\xa2\x02\n\x07\x46usMesh\x12\x19\n\x08Vertices\x18\x01 \x03(\x0b\x32\x07.float3\x12\x12\n\x06\x43olors\x18\x02 \x03(\rB\x02\x10\x00\x12\x18\n\x07Normals\x18\x03 \x03(\x0b\x32\x07.float3\x12\x14\n\x03UVs\x18\x04 \x03(\x0b\x32\x07.float2\x12\x1c\n\x0b\x42oneWeights\x18\x05 \x03(\x0b\x32\x07.float4\x12\x1c\n\x0b\x42oneIndices\x18\x06 \x03(\x0b\x32\x07.float4\x12\x15\n\tTriangles\x18\x07 \x03(\rB\x02\x10\x00\x12\x1b\n\x0b\x42oundingBox\x18\x08 \x01(\x0b\x32\x06.AABBf\x12\x19\n\x08Tangents\x18\t \x03(\x0b\x32\x07.float4\x12\x1b\n\nBiTangents\x18\n \x03(\x0b\x32\x07.float3\x12\x10\n\x08MeshType\x18\x0b \x01(\x05\"K\n\x07\x46usNode\x12\x0c\n\x04Name\x18\x01 \x01(\t\x12\x16\n\nComponents\x18\x02 \x03(\x05\x42\x02\x10\x00\x12\x1a\n\x08\x43hildren\x18\x03 \x03(\x0b\x32\x08.FusNode\"L\n\x08\x46usScene\x12$\n\rComponentList\x18\x01 \x03(\x0b\x32\r.FusComponent\x12\x1a\n\x08\x43hildren\x18\x02 \x03(\x0b\x32\x08.FusNode\"_\n\x0c\x46usTransform\x12\x1c\n\x0bTranslation\x18\x01 \x01(\x0b\x32\x07.float3\x12\x19\n\x08Rotation\x18\x02 \x01(\x0b\x32\x07.float3\x12\x16\n\x05Scale\x18\x03 \x01(\x0b\x32\x07.float3\"t\n\tFusWeight\x12$\n\tWeightMap\x18\x01 \x03(\x0b\x32\x11.VertexWeightList\x12\x1d\n\x06Joints\x18\x02 \x03(\x0b\x32\r.FusComponent\x12\"\n\x0f\x42indingMatrices\x18\x03 \x03(\x0b\x32\t.float4x4\"6\n\x10NormapMapChannel\x12\x0f\n\x07Texture\x18\x01 \x01(\t\x12\x11\n\tIntensity\x18\x02 \x01(\x02\"6\n\x0fSpecularChannel\x12\x11\n\tShininess\x18\x01 \x01(\x02\x12\x10\n\x08Strength\x18\x02 \x01(\x02\"2\n\x0cVertexWeight\x12\x12\n\nJointIndex\x18\x01 \x01(\x05\x12\x0e\n\x06Weight\x18\x02 \x01(\x02\"8\n\x10VertexWeightList\x12$\n\rVertexWeights\x18\x01 \x03(\x0b\x32\r.VertexWeight\"\x1e\n\x06\x66loat2\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\")\n\x06\x66loat3\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\x12\t\n\x01z\x18\x03 \x01(\x02\"4\n\x06\x66loat4\x12\t\n\x01x\x18\x01 \x01(\x02\x12\t\n\x01y\x18\x02 \x01(\x02\x12\t\n\x01z\x18\x03 \x01(\x02\x12\t\n\x01w\x18\x04 \x01(\x02\"f\n\x08\x66loat4x4\x12\x15\n\x04Row1\x18\x01 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row2\x18\x02 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row3\x18\x03 \x01(\x0b\x32\x07.float4\x12\x15\n\x04Row4\x18\x04 \x01(\x0b\x32\x07.float4*\x17\n\x0c\x44istribution\x12\x07\n\x03GGX\x10\x00*\x1f\n\x08LerpType\x12\x08\n\x04Lerp\x10\x00\x12\t\n\x05Slerp\x10\x01*:\n\tLightType\x12\t\n\x05Point\x10\x00\x12\x0c\n\x08Parallel\x10\x01\x12\x08\n\x04Spot\x10\x02\x12\n\n\x06Legacy\x10\x03*5\n\x10ProjectionMethod\x12\x0f\n\x0bPerspective\x10\x00\x12\x10\n\x0cOrthographic\x10\x01*V\n\x06TypeId\x12\n\n\x06\x44ouble\x10\x00\x12\x07\n\x03Int\x10\x01\x12\t\n\x05\x46loat\x10\x02\x12\n\n\x06\x46loat2\x10\x03\x12\n\n\x06\x46loat3\x10\x04\x12\n\n\x06\x46loat4\x10\x05\x12\x08\n\x04\x42ool\x10\x06\x62\x06proto3')

_DISTRIBUTION = DESCRIPTOR.enum_types_by_name['Distribution']
Distribution = enum_type_wrapper.EnumTypeWrapper(_DISTRIBUTION)
_LERPTYPE = DESCRIPTOR.enum_types_by_name['LerpType']
LerpType = enum_type_wrapper.EnumTypeWrapper(_LERPTYPE)
```

The error occurs at the statement `_DISTRIBUTION = DESCRIPTOR.enum_types_by_name['Distribution']`

**Comment by griestopf** [comment]
@ASPePeX : any idea why the CI build yields a different FusSerialization_pb2.py than the build on the dev machine. Some protobuf (compiler) version mismatch between Linux and Windows?

**Comment by ASPePeX** [comment]
We have duplicate references in the project file. There might be a difference in precedence.

Hopefully this will fix it af257fb2f02c6bdc23b1d35a0a6c4b720ee02222

**Comment by ASPePeX** [comment]
That was it, I reuploaded the working addon to the release page.
